client/src/components/PhotoItem/PhotoItem.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const PhotoItem = (props) => {
```

Comment:
Not sure this needs its own component. Will there be more added to this? Seems to me as if this could be nested into another component.

Reply:
This will be used multiple times in both the grid component and within the modal.